exporter/model.go

```diff
@@ -57,11 +57,12 @@ type importable struct {
 }
 
 type reference struct {
-	Path     string
-	Resource string
-	Match    string
-	Variable bool
-	File     bool
+	Path      string
+	Resource  string
+	Match     string
+	MatchType string // type of match, `prefix` - reference is embedded into string, `` (or `exact`) - full match
```

Comment:
```suggestion
	MatchPrefix bool
```
Wouldn't a simple flag here be simpler? is there a third match type we have in mind to justify a new category?

Reply:
I'm thinking right now about additional  match types:
* `substring` - secret scope references
* `suffix` - for workspace support in SQL queries/dashboards.